eks/outputs.tf

```diff
@@ -1,11 +1,14 @@
 output "id" {
-  value = module.thanos.id
+  description = "ID to pass other modules in order to refer to this module as a dependency. It takes the ID that comes from the main module and passes it along to the code that called this variant in the first place."
+  value       = module.thanos.id
 }
 
 output "metrics_archives" {
-  value = local.metrics_archives
+  description = "Bucket configuration in an HCL structure that will be used as an output to pass on to the module `kube-prometheus-stack` to activate and then configure `thanos-sidecar`."
+  value       = local.metrics_archives
 }
 
 output "thanos_enabled" {
-  value = module.thanos.thanos_enabled
+  description = "Flag to say that Thanos is enabled. It takes the output that comes from the main module and passes it along to the code that called this variant in the first place."
```

Comment:
I would say only "Boolean indicating wether Thanos is enabled." 
I am not sure what is the interest of having this value though,but I guess that discussion is not related to your PR